docs/rules/valid-package-definition.md

```diff
@@ -45,6 +45,25 @@ Examples of **correct** code for this rule:
 }
 ```
 
+### Options
+
+Pass an array of top-level package properties to ignore.
+When provided, any errors related to the properties, won't cause the rule to report a violation.
+This can be useful if you're using any of the more granular `valid-*` rules alongside this one.
+That way you're not double-reporting violations.
+
+Example of excluding the bin property:
+
+```json
+{
+	"package-json/valid-bin": "error",
+	"package-json/validate-package-definition": [
+		"error",
+		{ "ignoreProperties": ["bin"] }
+	]
+}
+```
+
```

Comment:
For now, the default `recommended` config sets `ignoreProperties` to ignore those flagged by the newer, more granular `valid-*` rules.
Once all `valid-*` rules are implemented then `valid-package-definition` will be removed from `recommended`.
See [#850 💬 Discussion: Which require- and valid- rules should be in the recommended config?](https://github.com/JoshuaKGoldberg/eslint-plugin-package-json/issues/850) for the list of rules that will be added.